Recurly.Tests/MockClient.cs

```diff
@@ -45,7 +45,7 @@ internal static MockClient Build<T>(Dictionary<Func<IRestRequest, bool>, Mock<IR
                     .Returns(item.Value.Object);
 
                 mockIRestClient
-                    .Setup(x => x.ExecuteTaskAsync<T>(It.Is<IRestRequest>(r => item.Key(r)), It.IsAny<CancellationToken>()))
+                    .Setup(x => x.ExecuteAsync<T>(It.Is<IRestRequest>(r => item.Key(r)), It.IsAny<CancellationToken>()))
```

Comment:
Is this to fix the deprecation warning? The are equivalent behavior?

Reply:
Yes, this is to fix the deprecation warning and has the same behavior